advanced/device-allowednetworks.rst

```diff
@@ -1,4 +1,4 @@
-.. _allowed-networks:
+.. _advanced-device-allowednetworks:
```

Comment:
Could all these reference targets for specific options be replaced by a direct link to the .rst file?
Sphinx seems to support such links, and it would ease maintenance: https://stackoverflow.com/a/54127156/9294540